Userland/Libraries/LibCrypto/Cipher/AES.cpp

```diff
@@ -41,26 +47,15 @@ ByteString AESCipherKey::to_byte_string() const
 }
 #endif
 
-void AESCipherKey::expand_encrypt_key(ReadonlyBytes user_key, size_t bits)
+#if (ARCH(I386) || ARCH(X86_64))
+static void __attribute__((target("default"))) AESCipherKey_expand_encrypt_key(u32* round_key, ReadonlyBytes user_key, size_t bits)
+#else
```

Comment:
With #24668 this pattern might change, although not sure if it affects the parts of ifuncs that actually seem work

Reply:
That change looks very good, great work of Dan!